nengo/simulator.py

```diff
@@ -55,11 +55,12 @@ def __len__(self):
 class Simulator(object):
     """Reference simulator for Nengo models."""
 
+    # 'unsupported' defines features unsupported by a simulator.
+    # The format is a list of tuples of the form `(test, reason)` with `test`
+    # being a regular expression matched against Nengo test names, and `reason`
+    # is a string describing why the feature is not supported by the backend.
+    # E.g., unsupported = [('test_pes*', 'PES rule not implemented')]
```

Comment:
I like option 1 because it gives the most control, but I'm also used to having to put those stars myself. I don't like adding the star to the end in option 2; I feel like we have a lot of tests like `test_something` and `test_something_else`, and I might only want to match the first one (actually this is one thing I don't like about the way `-k` works on `py.test`). But I don't think there's really any downside to automatically adding a star to the beginning, and it might be a good idea to add it automatically. I'm against option 3 for the same reason as option 2.

So I'd be fine with option 1 or modified option 2 only adding the star to the beginning.